src/app/public/modules/tabs/tab-button.component.html

```diff
@@ -29,7 +33,7 @@
       [disabled]="disabled"
       [attr.aria-label]="'skyux_tab_close' | skyLibResources"
       *ngIf="allowClose"
-      (click)="doCloseClick();$event.stopPropagation()"
+      (click)="doCloseClick($event)"
```

Comment:
Pulled these out of the template and into the component so that we could provide proper comments about _why_ they're needed.